src/main/resources/application-tenants.yml

```diff
@@ -3,9 +3,10 @@ bpmns:
     - id: "rhino"
       flows:
         payment-transfer: "minimal_mock_fund_transfer-{dfspid}"
-    - id: "lion"
-      flows:
-        payment-transfer: "MockPayerFundTransfer-{dfspid}"
     - id: "gorilla"
+      flows:
+        payment-transfer: "minimal_mock_fund_transfer_account_lookup-{dfspid}"
+        outbound-transfer-request: "mpesa_flow_{ams}-{dfspid}"
+    - id: "lion"
```

Comment:
Please do not change default from here, it will break backward compatibility. Please make sure you only add the new config and not change tenant mapping for existing ones. Gorilla tenant, payment transfer should be PayerFundTransfer not minimally mock. For making changes in tenant to orchestration mapping use helm chart level configurations